include/istio/control/http/controller.h

```diff
@@ -81,6 +84,8 @@ class Controller {
     // The LRU cache size for service config.
     // If not set or is 0 default value, the cache size is 1000.
     int service_config_cache_size{};
+
+    const ::istio::utils::LocalNode local_node;
```

Comment:
Options is transit, just used to pass data.  so you should just store LocalNode& as reference here.